examples/jdbc2-complex-type-queries/jdbc2_complex_type_queries.description

```diff
@@ -0,0 +1,6 @@
+// This example demonstrates how to use the JDBC client with complex data types such as arrays,
```

Comment:
The name of the .description file should be the name of the directory with `-`s replaced by `_`s.

jdbc2_complex_type_queries.description

Since we have only one bal file in these examples, shall we update the bal and out file names similarly too?

Reply:
I think the names are correct here. Directory name : jdbc2-complex-type-queries, and files names are: jdbc2_complex_type_queries.bal, jdbc2_complex_type_queries.description, jdbc2_complex_type_queries.out. The files names are replaced '-' by '_'.
Is there anything wrong in the above names? Sorry, it's not clear for me what change you wanted to have.

Reply:
Yeah, sorry my bad, I seem to have misread something. This is correct.